src/Entity/Transformer.php

```diff
@@ -181,8 +181,8 @@ public function loadBlock(array $content, PageInterface $page): PageBlockInterfa
 
         $block->setEnabled($content['enabled']);
 
-        if (isset($content['position'])) {
-            $block->setPosition($content['position']);
+        if (isset($content['position']) && is_numeric($content['position'])) {
```

Comment:
Should we throw an exception when it's set but not a numeric ?

Reply:
Not sure TBH, old snapshot data, if it is not easily fixable, might contains some weird data.

Reply:
For now I would go for the , let's try to see what data can get on the snapshot content and fix all the errors. Later we might want to provide a command to fix snapshots, or fix snapshots on the fly while being loaded. Not sure right now, but at a first step, I would prefer to not throw a lot of exceptions on old data.

Reply:
Yeah sure. I think you can just cast to int everytime and it's ok for me